provision-contest/ansible/roles/system_fixes/tasks/main.yml

```diff
@@ -65,5 +65,13 @@
         line: 'DNS=8.8.8.8'
       notify: Reboot the machine
 
+- name: stat /etc/cron.daily/etckeeper
+  stat: path=/etc/cron.daily/etckeeper
+  register: daily_etckeeper
+
+- name: Enable etckeeper autocommits every hour
+  command: mv /etc/cron.daily/etckeeper /etc/cron.hourly/etckeeper
```

Comment:
I rather see a symlink here (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) as it removes the register step and gives us the benefits of ansible.

Reply:
Can you just commit the code? I am just cherry-picking code written by others